ios/Classes/PTFlutterDocumentController.h

```diff
@@ -18,14 +18,14 @@ NS_ASSUME_NONNULL_BEGIN
 // viewer options
 @property (nonatomic, copy, nullable) NSArray<NSString *> * annotationToolbars;
 @property (nonatomic, copy, nullable) NSArray<NSString *> * hideDefaultAnnotationToolbars;
-@property (nonatomic, assign) BOOL hideAnnotationToolbarSwitcher;
-@property (nonatomic, assign) BOOL hideTopToolbars;
-@property (nonatomic, assign) BOOL hideTopAppNavBar;
-@property (nonatomic, assign) BOOL showNavButton;
+@property (nonatomic, assign, getter=isAnnotationToolbarSwitcherHidden) BOOL annotationToolbarSwitcherHidden; // hideAnnotationToolbarSwitcher configuration option
+@property (nonatomic, assign) BOOL topToolbarsHidden; // hideTopToolbars configuration option
+@property (nonatomic, assign) BOOL topAppNavBarHidden; // hideTopAppNavBar configuration option
+@property (nonatomic, assign) BOOL navButtonShown; // showNavButton configuration option
```

Comment:
I would keep this one as it was. Not keeping with iOS conventions but I'm not sure that's really possible for this one, and "Shown" seem odd..